docs/source/getting_started/using_entitysets.ipynb

```diff
@@ -0,0 +1,337 @@
+    "\n",
+    "* The ``index`` parameter specifies the column that uniquely identifies rows in the dataframe.\n",
+    "* The ``time_index`` parameter tells Featuretools when the data was created.\n",
+    "* The ``logical_types`` parameter indicates that \"product_id\" should be interpreted as a Categorical column, even though it is just an integer in the underlying data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from woodwork.logical_types import Categorical, PostalCode\n",
+    "\n",
+    "es = es.add_dataframe(\n",
+    "    dataframe_name=\"transactions\",\n",
+    "    dataframe=transactions_df,\n",
+    "    index=\"transaction_id\",\n",
+    "    time_index=\"transaction_time\",\n",
+    "    logical_types={\n",
+    "        \"product_id\": Categorical,\n",
+    "        \"zip_code\": PostalCode,\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "es"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. currentmodule:: featuretools\n",
+    "\n",
+    ".. note ::\n",
+    "\n",
+    "    You can also display your entity set structure graphically by calling :meth:`.EntitySet.plot`.\n",
+    "\n",
+    "This method associates each column in the dataframe to a logical type. If you don't specify the logical type for a column, it gets inferred based on the underlying data. The logical types of a dataframe are listed in the schema."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es[\"transactions\"].ww.schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we can do that same thing with our products dataframe."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es = es.add_dataframe(\n",
+    "    dataframe_name=\"products\",\n",
+    "    dataframe=products_df,\n",
+    "    index=\"product_id\")\n",
+    "\n",
+    "es"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With two dataframes in our entity set, we can add a relationship between them.\n",
+    "\n",
+    "## Adding a Relationship\n",
+    "\n",
+    "We want to relate these two dataframes by the columns called \"product_id\" in each dataframe. Each product has multiple transactions associated with it, so it is called it the **parent dataframe**, while the transactions dataframe is known as the **child dataframe**. When specifying relationships we list the columns in the parent dataframe first. Note that each `ft.Relationship` must denote a one-to-many relationship rather than a relationship which is one-to-one or many-to-many."
```

Comment:
I think this needs updated a bit. Before we just specified parent variable and child variable as parameters when adding the relationship, but now we need to pass four parameters...parent dataframe name, parent column name, child dataframe name and child column name. In particular, the third sentence needs to be updated to reflect this change instead of just referring to columns.

Reply:
✅ f5fcc78f